swagger_server/config.ini.template

```diff
@@ -5,3 +5,15 @@
 
 EVALUATION_TYPE=dryrun
 SCENARIO_DIRECTORY=swagger_server/itm/data/%(EVALUATION_TYPE)s/scenarios/
+
+ADEPT_PORT=8081
+ADEPT_HOSTNAME=
+
+SOARTECH_PORT=8084
+SOARTECH_HOSTNAME=
+
+SAVE_HISTORY=False
+HISTORY_DIRECTORY=itm_history_output
+
+SAVE_HISTORY_TO_S3=False
+HISTORY_S3_BUCKET=itm-ui-assets
```

Comment:
BTW, this isn't high-priority-- or at least, I wouldn't want to hold up this ticket for it, because I'd like to get this ticket in to the upcoming release (which hopefully will be today or maybe tomorrow at the latest).  If it doesn't have a quick/easy solution, we can defer it.

Reply:
Ok lets defer, I'll just commit the changes to remove the environment variables